src/glossary.md

```diff
@@ -17,6 +17,7 @@ ICE                     |  internal compiler error. When the compiler crashes.
 ICH                     |  incremental compilation hash. ICHs are used as fingerprints for things such as HIR and crate metadata, to check if changes have been made. This is useful in incremental compilation to see if part of a crate has changed and should be recompiled.
 infcx                   |  the inference context (see `librustc/infer`)
 MIR                     |  the Mid-level IR that is created after type-checking for use by borrowck and trans ([see more](./mir.html))
+miri                    |  an interpreter for MIR used for constant evaluation ([see more](./miri.html))
```

Comment:
We've never capitalized it so far, just like `rustc` is never capitalized, but maybe we should?
@nikomatsakis @solson @oli-obk Opinions?

Reply:
I've always used "miri" and "Miri", never "MIRI", though I guess that's an arbitrary aesthetic choice.

It particularly makes sense to go with lowercase `miri` when you're talking about a binary (like `rustc`).

Reply:
It originated as an acronym though, so probably makes sense to capitalise it, even if we write it like an ordinary word in informal contexts.

Reply:
It doubles as an acronym but was also named because it coincides with a human name. Plus writing it in all caps would be inconsistent with all existing documentation.

Reply:
There's also that MIRI collides with [another thing entirely](https://intelligence.org/)

Reply:
I prefer "miri" to MIRI. I'm hard put to justify this, but I guess that to me it doesn't feel like an "acronym", more like a "thing".